File: source/aws/ts/lambda-docker-websocket/lambda_entry.ts
import './use_env';
import {} from 'aws-sdk';
import { IWebsocketEvent } from './UniRpc/LambdaWebsocketTypes';
import { WebsocketService } from './UniRpc/WebsocketService';

export async function handler (event: IWebsocketEvent) {
    console.log('AWS Lambda Project "@{project}".');
    let websocketService = new WebsocketService();
    // Register your service implementations here.
    // websocketService.RegisterService(new Namespace.ServiceImpl());
    return await websocketService.ProcessEvent(event);
};

// Entry point fo the process
if (require.main == module) {
    (async () => await handler({} as any))();
}

File: source/aws/py/lambda-docker-websocket/lambda_entry.py
# use the '../.env' environment variables for local debugging
import use_env
import json
import asyncio
from typing import Dict
from UniRpc.LambdaWebsocketTypes import IWebsocketEvent
from UniRpc.WebsocketService import WebsocketService


async def websocket_handler(event: IWebsocketEvent):
    websocketService: WebsocketService = WebsocketService()
    # register your service implementations here
    # websocketService.RegisterService(Namespace.ServiceImpl())
    response = await websocketService.ProcessEvent(event)
    print('Function Response:', json.dumps(response))
    return response


def handler(event: IWebsocketEvent, context: Dict[str, any] = None):
    return asyncio.run(websocket_handler(event))


if __name__ == '__main__':
    handler({})